frontend/client/views/secrets/index.vue

```diff
@@ -464,9 +473,12 @@ export default {
       })
     },
 
-    deleteSecret: function () {
+    deleteSecret: function (path) {
```

Comment:
Good catch on the parameter. I don't know what possessed me to make this function use `this.currentPath` instead of a parameterized path. I'm a fan of pure functions when possible.

Let's just make this function take a path and try to delete it as a secret. The second button (red garbage bin) should trigger this function with the `currentPath`